### DISCUSSION TITLE
Hoe gaan we om met Rollen in de taxonomie van CB-NL

### DISCUSSION THREAD

**Discussion by NielsHoffmann** [bimloket/CB-NL#14] [General]
Hoe gaan we om met Rollen in de taxonomie van CB-NL

**Comment by hennystolwijk** [comment]
Wat mij betreft allen in de taxonomie van het Kernmodel als mogelijk 'haakje', omdat dit in NEN2660 is gedefinieerd. 
Verder niet in de taxonomie gebruiken.
